istio-telemetry/kiali-operator/templates/kiali.yaml

```diff
@@ -0,0 +1,425 @@
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali
+  namespace: {{ .Release.Namespace }}
+  labels:
+    release: {{ .Release.Name }}
+annotations:
+  ansible.operator-sdk/reconcile-period: "0s"
+spec:
+  ###################################################################
+  # kiali_cr.yaml
+  #
+  # This is a fully documented Kiali custom resource yaml file.
```

Comment:
Personally, I would not include all of these comments in here. This is a copy of the kiali_cr located in the kiali project:

https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml

As Kiali adds/modifies/removes things in here, these comments are going to change as that file and its comments are used for documentation as well as being an example CR.

But this means this istio copy of the file's comments are going to have to be modified to track those changes constantly. I am fairly confident the upstream kiali_cr file and this file are going to go out of sync quickly.

Perhaps just have a link to the kiali_cr.yaml file for the version of kiali being used by Istio... for example, if you are pulling in Kiali 1.0, have a simple small comment at the top to say:

"For details about the supported settings, see https://github.com/kiali/kiali/blob/v1.0/operator/deploy/kiali/kiali_cr.yaml"

Just my $0.02

Reply:
agree with your idea.  I will make the change according to your comments. Thanks.